.ci/integration_test.sh

```diff
@@ -95,20 +106,23 @@ get_build_chain()
 }
 
 precompiled_test(){
-    if [ -z "$(go test -v ./precompiled/cns| grep FAIL)";then LOG_ERROR "test precompiled/cns failed" && exit 1; fi
-    if [ -z "$(go test -v ./precompiled/config| grep FAIL)";then LOG_ERROR "test precompiled/config failed" && exit 1; fi
-    if [ -z "$(go test -v ./precompiled/consensus| grep FAIL)";then LOG_ERROR "test precompiled/consensus failed" && exit 1; fi
-    if [ -z "$(go test -v ./precompiled/crud| grep FAIL)";then LOG_ERROR "test precompiled/crud failed" && exit 1; fi
-    if [ -z "$(go test -v ./precompiled/permission| grep FAIL)";then LOG_ERROR "test precompiled/permission failed" && exit 1; fi
+# TODO: consensus test use getSealer first
+    precompileds=(config cns crud permission)
+    for pkg in ${precompileds[*]}; do
+        if [ ! -z "$(go test -v ./precompiled/${pkg}| grep FAIL)" ];then 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=4970769)